rbac/server/api/roles.py

```diff
@@ -100,14 +97,21 @@
                 "name": role_title,
                 "remote_id": distinguished_name_formatted,
             }
+            env = Env()
+            if env.int("ENABLE_LDAP_SYNC", 0):
```

Comment:
We currently are not supporting multiple providers at one time correct? If so, you can resolve this comment. 

If not, this would cause issues if we have Azure and LDAP turned on at the same time and we want to send this newly created role to both external providers.